comps/rerankings/src/Dockerfile

```diff
@@ -36,6 +36,7 @@ fi && \
 pip install --no-cache-dir --upgrade pip setuptools && \
```

Comment:
https://github.com/opea-project/GenAIComps/actions/runs/13555365160/job/37926867062?pr=1318
![image](https://github.com/user-attachments/assets/12c46742-e750-48a5-a21d-35b67c0ae842)   
Requirement file name not correct.